src/app/static/src/app/vue-chart/src/bar/BarChartWithFilters.vue

```diff
@@ -224,17 +224,17 @@
                 return toFilterLabelLookup(this.selections.selectedFilterOptions[this.selections.xAxisId] || []);
             },
             filtersAsOptions() {
-                return this.filterConfig.filters.map((f: Filter) => ({id: f.id, label: f.label}))
+                return this.filterConfig.filters.map((f: Filter) => ({id: f.id, label: f.label})) as FilterOption[]
```

Comment:
Missing test for this type cast. Can it not be in the method signature as `filtersAsOptions(): FilterOption[] {}` ?